internal/runners/packages/install.go

```diff
@@ -1,27 +1,74 @@
 package packages
 
 import (
+	"github.com/ActiveState/cli/internal/analytics"
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/prompt"
+	"github.com/ActiveState/cli/internal/runbits/requirements"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
 	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/project"
 )
 
+type PackageVersion struct {
+	captain.NameVersion
+}
+
+func (pv *PackageVersion) Set(arg string) error {
+	err := pv.NameVersion.Set(arg)
+	if err != nil {
+		return locale.WrapInputError(err, "err_package_format", "The package and version provided is not formatting correctly, must be in the form of <package>@<version>")
+	}
+	return nil
+}
+
 // InstallRunParams tracks the info required for running Install.
 type InstallRunParams struct {
 	Package PackageVersion
 }
 
 // Install manages the installing execution context.
 type Install struct {
-	prime primeable
```

Comment:
Sorry.. I didn't realize that using prime in a non-standard way was a pre-existing issue rather than one you introduced.